proto/process/agent.proto

```diff
@@ -672,6 +675,8 @@ message Connection {
 	ConnectionDirection direction = 19;
 	uint64 lastPacketsSent = 38;
 	uint64 lastPacketsReceived = 39;
+	ProtocolStack protocol = 48;
```

Comment:
Would a bitfield be better here? It would save on some memory.

Reply:
we can have a lot of protocols, but with bitfield we will be limited to 16 protocols (or 32, or 64)
i think we should not limit ourselves for such a low number of protocols 
especially when we aim to classify and parse aws and google protocols which adds us a lot more protocols